client/src/Components/SingularPost.js

```diff
@@ -18,6 +18,20 @@ export default function SingularPost(props) {
     setOpen(false);
```

Comment:
Whoa, not part of your PR, but this method didn't belong here, it should be on Profile